rcl/test/rcl/test_graph.cpp

```diff
@@ -263,7 +264,7 @@ check_graph_state(
     rcl_reset_error();
     is_in_tnat = false;
     for (size_t i = 0; RCL_RET_OK == ret && i < tnat.topic_count; ++i) {
-      if (topic_name == std::string(tnat.topic_names[i])) {
+      if (fq_topic_name == tnat.topic_names[i]) {
```

Comment:
Connext might be failing due to these changes. It might be related to the bug @clalancette is running into with the bridge and needs a fixup from @Karsten1987, which he is working on now.